src/views/terms-and-conditions.html

```diff
@@ -0,0 +1,96 @@
+            </li>
+        </ul>
+    </div>
+</div>
+<div class="seven half unit row">
+    <ul>
+        <li>
+            Except as required by law, the Organisers of Hack Cambridge, or the venue owners Cambridge Live trust, will not be held liable for any property loss or damage, personal injury or death, or any economic loss, howsoever caused, including by the negligence of the Organisers or Cambridge Live. These aforementioned losses or damages or injuries or deaths include those arising directly or indirectly from the Event or any other aspect related thereto or in connection with the Event or this Agreement.
+        </li>
+        <li>
+            The Organisers or Cambridge Live will not be held liable for any property damage or loss, personal injury or death arising from the behaviour or actions of any attendees of the Event. Attendees can include, but are not limited to volunteers, representatives from any companies, mentors, judges, room hosts or the Organisers themselves. This includes behaviour or actions before, during, after the Event, or are in any way connected to the Event.
+        </li>
+        <li>
+            The Organisers give no warranties in respect of any aspect of the Event or any materials related thereto or offered at the Event and, to the fullest extent possible under the laws governing this Agreement, disclaims all implied warranties, including but not limited to warranties of fitness for a particular purpose, accuracy, timeliness, and merchantability. The Event is provided on an “as-is” basis. The Organisers do not accept any responsibility or liability for reliance by you or any person on any aspect of the Event or any information provided at the Event.
+        </li>
+        <li>
+            Each attendee must take reasonable steps to ensure their own safety and the safety of others, as well as take reasonable care of the property of theirselves and others. This especially applies to circumstances where they are tired, when they are working with hardware and potentially dangerous equipment, as well as when they are consuming any food or drink. They will check that the food or drink complies with their dietary requirements, if any.
+        </li>
+    </ul>
+</div>
+<div class="one unit row">
+    <h3>Reimbursements</h3>
+</div>
+<div class="four unit row">
+    <div class="columns balance">
```

Comment:
Do you have a reason for splitting this part into columns when the above is block?

Reply:
It's my attempt to differentiate what appear to be different bits of the terms and conditions. In particular, what appears (to me) to be mainly stock legalese is block text, and the more particular information is in columns. I think the reduced line length of the columns also improves readability. However, this could really do with headers—is there a legal requirement not to make the stuff in red (`class="secondary"`) a header? Sections would also be nice—I can envisage this section being red for example, since it appears to be important.

Reply:
I don't think there's legal issues with converting the `class="secondary"` elements to headers, but perhaps @moosichu could confirm?

I agree, I think including headers in the columns or making this section a different colour would make it more clear that we're highlighting this particular part of the terms.